package.json

```diff
@@ -1,16 +1,20 @@
 {
   "name": "business-filings-ui",
-  "version": "7.3.27",
+  "version": "7.4.0",
   "private": true,
   "appName": "Filings UI",
   "sbcName": "SBC Common Components",
   "scripts": {
     "dev": "vite",
     "build": "vite build",
-    "preview": "vite preview",
-    "lint": "eslint . --ext .js,.ts,.vue",
+    "build-check": "vite build",
+    "preview": "vite preview --port 8080",
+    "lint": "eslint . --ext js,ts,vue --no-fix",
+    "lint:fix": "eslint . --ext js,ts,vue --fix src",
```

Comment:
I don't want `npm run lint` to automatically fix issues, so I created "lint:fix" to do that.

This is different from auth-web where the default is to fix and there is "lint:nofix" instead.